Core/PARStore.m

```diff
@@ -1458,8 +1468,11 @@ - (void)_sync
     
     // just go through each row (back in time) until all entries are loaded
     NSMutableDictionary *updatedValues = [NSMutableDictionary dictionary];
-    for (NSManagedObject *log in allLogs)
+    for (NSManagedObjectID *logID in allLogObjectIDs)
     {
+        // Get object
+        NSManagedObject *log = [moc objectWithID:logID];
```

Comment:
Yeah, my understanding is that even if you fetch just objectIDs, the row cache gets populated, and so forming objects with `objectWithID:` should be fast. (I use this a lot in Ensembles)